vimeo/VimeoUpload
# Updates framework to support changes in VimeoNetworking

### Implementation Summary
Updated VimeoNetworking to pull in the latest changes from its `develop` branch, which includes changes from the 3.1.0 release. 

This required creating a new static constant in **NSURL+Upload.swift** to replace the `VimeoBaseURL` constant in VimeoNetworking which is no longer accessible outside of the module. The new constant looks like this:

```Swift
static let VimeoBaseURL = URL(string: "https://api.vimeo.com")!
```

Other changes:
- The **Podfile** was updated to ensure VimeoNetworking is build using Swift 4.
- Fixed a build error in **DemoCameraRollCell.swift** where we were trying to call a static method on `String` which didn't seem to exist. The method does, however, exist on `NSString`.


### How to Test
- Ensure upload behaves as expected. There should be no feature or functional changes here.
